test/alternator/test_streams.py

```diff
@@ -15,6 +15,13 @@
 from urllib.error import URLError
 from boto3.dynamodb.types import TypeDeserializer
 
+# All tests in this file are expected to fail with tablets due to #16317.
+# To ensure that Alternator Streams is still being tested, instead of
+# xfailing these tests, we temporarily coerce the tests below to avoid
+# using default tablets setting, even if it's available. We do this by
+# using the following tags when creating each table below:
+TAGS = [{'Key': 'experimental:initial_tablets', 'Value': 'none'}]
```

Comment:
Btw, in 49026dc319f55769d3c20719aa75cccc215e91be Pavel added auto-calculation of initial tablet count count, and the user just specifies that the keyspace should be tablet-aware. Do we want to do the same for alternator?

Reply:
Yes, the main goal of this PR wasn't to fix the tests (this was a necessary part) but to make using tablets in Alternator automatic, like we do in CQL:
1. After this patch, if the "tablets" experimental feature is on, all Alternator tables will use tablets by default, with the default initial_tablets calculation you have in CQL (I pass initial_tablets=0 to the lower-level code).
2. The user is given an experimental way to choose a different initial_tablets or to disable tablets completely.
3. There is no API to specify that the table **should** use tablets, because it's already on by default.